src/fswAlgorithms/orbitControl/lambertSolver/lambertSolver.h

```diff
@@ -0,0 +1,91 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+*/
+
+
+#ifndef LAMBERTSOLVER_H
+#define LAMBERTSOLVER_H
+
+#include "architecture/_GeneralModuleFiles/sys_model.h"
+#include "architecture/msgPayloadDefC/LambertProblemMsgPayload.h"
+#include "architecture/msgPayloadDefC/LambertSolutionMsgPayload.h"
+#include "architecture/msgPayloadDefC/LambertPerformanceMsgPayload.h"
+#include "architecture/utilities/bskLogging.h"
+#include "architecture/messaging/messaging.h"
+#include "architecture/utilities/avsEigenSupport.h"
+#include "architecture/utilities/astroConstants.h"
+#include <vector>
+#include <array>
+
+/*! @brief This module solves Lambert's problem using either the Gooding or the Izzo algorithm.
+ */
+class LambertSolver: public SysModel {
+public:
+    LambertSolver();
+    ~LambertSolver();
+
+    void Reset(uint64_t currentSimNanos) override;
+    void UpdateState(uint64_t currentSimNanos) override;
+
+    ReadFunctor<LambertProblemMsgPayload> lambertProblemInMsg;          //!< lambert problem input message
+    Message<LambertSolutionMsgPayload> lambertSolutionOutMsg;           //!< lambert solution output message
+    Message<LambertPerformanceMsgPayload> lambertPerformanceOutMsg;     //!< lambert performance output message
+
+    BSKLogger bskLogger;                                                //!< -- BSK Logging
+
+    //!< [deg] minimum angle between position vectors such that they are not considered too aligned.
+    double alignmentThreshold = 1.0;
```

Comment:
make private variable, should have setter and getter for this variable.

Reply:
Added setters and getters for all module variables.